exporter/coralogixexporter/logs_client.go

```diff
@@ -82,28 +82,23 @@ func (e *logsExporter) pushLogs(ctx context.Context, ld plog.Logs) error {
 	for i := 0; i < rss.Len(); i++ {
 		resourceLog := rss.At(i)
 		appName, subsystem := e.config.getMetadataFromResource(resourceLog.Resource())
+		resourceLog.Resource().Attributes().PutStr("cx.application.name", appName)
+		resourceLog.Resource().Attributes().PutStr("cx.subsystem.name", subsystem)
```

Comment:
Potential worthwhile to make the keys constant values at the global scope of the package?

Mainly to avoid potential typos but it should decrease the require allocations needed.